Sources/StreamChatUI/Composer/ChatMenu/EmojiMenu/Cell/GiphyCollectionCell.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  CustomCell.swift
+//  StreamChat
+//
+//  Created by Mohammed Hanif on 27/04/22.
+//  Copyright © 2022 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+import StreamChatUI
+import StreamChat
+import GiphyUISDK
+import UIKit
+import Stipop
+
+class GiphyCollectionCell: UICollectionViewCell {
+
+    // MARK: - Variables
+    private var mediaView: GPHMediaView!
```

Comment:
@mdhanif-simformsolutions  Please avoid force-casting.

Reply:
Updated this. thank you.